Kentico.Kontent.Management/Models/Types/Elements/ValidationRegexModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Kentico.Kontent.Management.Models.Types.Elements
+{
+    /// <summary>
+    /// Specifies a regular expression pattern used to validate the text element's value.
+    /// </summary>
+    public class ValidationRegexModel
+    {
+        /// <summary>
+        /// Gets or sets the regular expression used for validation
+        /// </summary>
+        public string Regex { get; set; }
+
+        /// <summary>
+        /// Specifies regular expression flags that affect the search. Supports only case-insensitive therefore only allowed flag is 'i'.
```

Comment:
Re-confirm what are the supported values - docs says only `i`, but the KCL-7856 comment says different.

Reply:
I have spoken with the team and they confirm that the documentation is right.